app/reducers/topic.js

```diff
@@ -47,8 +32,9 @@ const topics = (
   action
 ) => {
   switch (action.type) {
-    case types.GET_TOPICS_SUCCESS:
-      return action.res.data;
+    case types.REQUEST_SUCCESS:
+      if (action.data) return action.data;
```

Comment:
This could arguably be placed in `client` and `server`. Rather than dispatch and check, we could check before dispatching.